roles/openstack-setup/tasks/flavors.yml

```diff
@@ -1,14 +1,23 @@
 ---
-- name: is m1.tiny undersized?
-  shell: mysql -e "select root_gb from nova.instance_types where name='m1.tiny';" | grep 10
-  failed_when: False
-  changed_when: False
-  register: resize_tiny_flavor
-  run_once: true
-
-- name: bump root disk size on m1.tiny
-  shell: mysql -e "update nova.instance_types set root_gb=10 where name='m1.tiny';"
-  when: resize_tiny_flavor|failed
+- name: Add flavors
+  os_nova_flavor:
+    name: '{{ item.name }}'
+    ram: '{{ item.ram | int }}'
+    disk: '{{ item.disk | int }}'
+    vcpus: '{{ item.vcpus | int }}'
+    flavorid: '{{ item.flavorid | default(omit) }}'
+    rxtx_factor: '{{ item.rxtx_factor | default(omit) }}'
+    ephemeral: '{{ item.ephemeral | default(omit) }}'
+    is_public: '{{ item.is_public | default(omit) }}'
+    swap: '{{ item.swap | default(omit) }}'
+    state: present
+    auth:
+      auth_url: "{{ endpoints.auth_uri  }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password  }}"
+  with_items: '{{ ironic.flavors | default([]) }}'
+  when: not ironic.enabled | default(False) | bool
```

Comment:
I don't think we have a `enabled` flag for nova, and I'm not really sure what the setting should look like to have ironic vs regular nova, but I think it all keys on the ironic.enabled. 

If the allinone has ironic enabled, that's going to create a non-functional cloud, so maybe it shouldn't enable it?